grammar.js

```diff
@@ -274,6 +275,11 @@ module.exports = grammar({
       $.identifier
     ),
 
+    global_statement: $ => seq(
+      'global',
+      commaSep1($.identifier)
+    ),
```

Comment:
I based this on https://docs.python.org/3/reference/grammar.html (Python 2's grammar specifies `global` statements in the same way). To me this reads as a comma separated list of identifiers. Does that seem accurate to you @maxbrunsfeld?